website/docs/docs/cloud-integrations/semantic-layer/gsheets.md

```diff
@@ -50,28 +50,24 @@ queryBuilder="/img/docs/dbt-cloud/semantic-layer/gsheets-query-builder.jpg"
 
 ## Using saved selections
 
-<p><span>Saved selections allow you to save the inputs you've created in the {props.type} <strong>Query Builder</strong> and easily access them again so you don't have to continuously build common queries from scratch. To create a saved selection:</span></p>
+Saved selections allow you to save inputs in the **Query Builder** to easily access them again so you don't have to continuously build common queries from scratch. To create a saved selection:
 
-<ol>
-  <li>Run a <span>query in the {props.type} <strong>Query Builder</strong>.</span></li>
-  <li>Save the selection by selecting the arrow next to the <strong>Query</strong> button and then select <strong>Query & Save Selection</strong>.</li>
-  <li>The application saves these selections, allowing you to view and edit them from the hamburger menu under <strong>Saved Selections</strong>.</li>
-</ol>
+- Run a query in the **Query Builder**.
+- Save the selection by selecting the arrow next to the **Query** button and then select **Query & Save Selection**.
+- The application saves these selections, allowing you to view and edit them from the hamburger menu under **Saved Selections**.
 
 <Lightbox src="/img/docs/dbt-cloud/semantic-layer/gsheets-query-builder.jpg" width="25%" title="Query and save selections in the Query Builder using the arrow next to the Query button." />
 
-<p>You can also make these selections private or public:</p>
+You can also make these selections private or public:
 
-<ul>
-  <li><strong>Public selections</strong> mean your inputs are available in the menu to everyone on the sheet.</li>
-  <li><strong>Private selections</strong> mean your inputs are only visible to you. Note that anyone added to the sheet can still see the data from these private selections, but they won't be able to interact with the selection in the menu or benefit from the automatic refresh.</li>
-</ul>
+- **Public selections** mean your inputs are available in the menu to everyone on the sheet.
+- **Private selections** mean your inputs are only visible to you. Note that anyone added to the sheet can still see the data from these private selections, but they won't be able to interact with the selection in the menu or benefit from the automatic refresh.
 
 ### Refreshing selections
 
-<p>Set your saved selections to automatically refresh every time you load the addon. You can do this by selecting <strong>Refresh on Load</strong> when creating the saved selection. When you access the addon and have saved selections that should refresh, you'll see "Loading..." in the cells that are refreshing.</p>
+Set your saved selections to automatically refresh every time you load the addon. You can do this by selecting **Refresh on Load** when creating the saved selection. When you access the addon and have saved selections that should refresh, you'll see "Loading..." in the cells that are refreshing.
```

Comment:
Hiya @mirnawong1 

I could be very wrong but should addon be add-on? Apologies if I'm incorrect.

Kind Regards
Natalie

Reply:
good question, since this is a google sheet integration, that's how they write it. i had to check that too!